README.md

```
@@ -1,47 +1,94 @@
 ```
 
-## 🛠️ Usage and examples
+## 📖 Table of Contents
+
+- [Movie Details](#movie)
+- [Search](#search)
+- [Creators](#creators)
+- [User Ratings](#user-ratings)
+- [User Reviews](#user-reviews)
+- [Docker Support](#-docker-support)
+- [Development](#-development)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)